tests/integration/src/test/java/org/mule/test/core/context/notification/processors/MessageProcessorNotificationPathTestCase.java

```diff
@@ -63,7 +65,7 @@ public void scopes() throws Exception {
     testFlowPaths("enricher", "/0", "/0/0", "/1", "/1/0", "/1/0/0", "/1/0/1");
     testFlowPaths("until-successful", "/0", "/0/0", "/1");
     testFlowPaths("until-successful-with-processor-chain", "/0", "/0/0", "/0/0/0", "/0/0/1", "/1");
-    testFlowPaths("until-successful-with-enricher", "/0", "/0/0", "/0/0/0", "/1");
+    testFlowPaths("until-successful-with-enricher", "/0", "/0/0", "/0/0/0", "/0/0/0/0", "/0/0/1", "/1");
```

Comment:
I had to modify a config file that was causing a test to be flaky. This change accounts for that change